### ISSUE TITLE
config file error

### ISSUE THREAD

**Issue by joeblew99** [tulir/mauirc-server#1]
I am trying this out. The code is very clear thanks..

I run the make package-prep: build to get the config in the right place, but it seems the config is formatted incorrectly. It does find the config file.

go 1.7

```
x-MacBook-Pro:mauirc-server apple$ ./mauirc-server --config=./package/etc/mauirc
[11:23:28 29.08.2016] [INFO] Initializing mauIRC Server 2.0.0
[11:23:28 29.08.2016] [FATAL] Failed to load config: json: cannot unmarshal string into Go value of type int

```

**Comment by tulir** [comment]
It seems that the example config had the IDENT port as a string even though the program parses it as an int. Fixed in f7fe4cb2ea023127c3c288e7ec88278a22a35ff0

**Comment by joeblew99** [comment]
@tulir thankyou.
